src/neuroglancer/sliceview/vector_graphics/backend.ts

```diff
@@ -35,21 +36,36 @@ export class PointChunk extends SliceViewChunk {
 
     let source = this.source;
 
-    this.systemMemoryBytes = source!.chunkBytes;
-    this.gpuMemoryBytes = source!.chunkBytes;
+    let chunkBytes: number = 0;
+    if (this.vertexPositions) {
+      chunkBytes = chunkBytes + this.vertexPositions!.buffer.byteLength; 
```

Comment:
Actually, this code for calculating the memory usage should be in downloadSucceeded, not here, since the data won't be available at the time this called.  The code in the existing initializeVolumeChunk method in VolumeChunk that set systemMemoryBytes and gpuMemoryBytes according to source!.spec.chunkBytes should in fact be removed, as it was just left there by mistake.  Originally the chunk manager code expected systemMemoryBytes to be set before a chunk was downloaded in order to ensure there would be sufficient free memory before starting the download, but that is no longer the case, since for variable size chunks it isn't possible to estimate the memory requirement ahead of time.